### DISCUSSION TITLE
Tries to format code in interactive window

### DISCUSSION THREAD

**Discussion by AxesAccess** [34j/vscode-black#18] [Q&A]
## Current Behavior

The extension fails with 

```Error: Invalid value for 'SRC ...': Path '/Interactive-1.interactive' does not exist.```

when running python code in interactive window. 

## Expected Behavior

Skip formatting code in interactive window.

## Steps to Reproduce the Problem

  1. Install Jupyter Notebook extension.
  2. Divide python code by cells with lines consisting of "# %%".
  3. Run cell in interactive window.

## Environment

- Version: v1.0.3
- Platform: Linux
- Node.js Version: v6.11.2

**Comment by 34j** [comment]
Thanks for your report. As this extension calls Black directly in the terminal, unfortunately, it is principally impossible to format a file that has not been saved.

**Reply by AxesAccess** [reply]
Is it possible to check if file exists before running black?
